include/tvSettingsODM.h

```diff
@@ -235,134 +149,6 @@ tvError_t GetWBctrl(char *inputSrc, char *colortemp, char *color, char *ctrl, in
  */
 tvError_t SaveColorTemperatureUser(tvRGBType_t rgbType, int value,int boost);
 
-/**
- * @brief Gets the current CMS default value. To be deprecated soon.
- * @note SOC vendors can stub these ODM functions and mark it as weak
- *
- * This function returns the default CMS value
- *
- * @param[out] component_type      - Component type. Valid value will be a member of ::tvComponentType_t
- *
- * @return int
- *
- * @retval 15                       - Luma default value
- * @retval 50                       - Hue or saturation default value
- * @retval -1                       - Input parameter is invalid
- *
- * @pre tvInit() should be called before calling this API
- */
- int GetCMSDefault(tvComponentType_t component_type);
```

Comment:
Are these expected to be remove? The API is not listed in the PR description.

Reply:
Yes, it is expected to be removed. It was missing from the PR description, but I have now added it.